Source/WebKit/Shared/WebCoreArgumentCoders.cpp

```diff
@@ -1399,14 +1399,14 @@ std::optional<Ref<WebCore::SharedBuffer>> ArgumentCoder<WebCore::SharedBuffer>::
     return std::nullopt;
 }
 
-#if ENABLE(SHAREABLE_RESOURCE) && PLATFORM(COCOA)
+#if ENABLE(SHAREABLE_RESOURCE)
 static ShareableResource::Handle tryConvertToShareableResourceHandle(const ScriptBuffer& script)
 {
     if (!script.containsSingleFileMappedSegment())
         return ShareableResource::Handle { };
 
     auto& segment = script.buffer()->begin()->segment;
-    auto sharedMemory = SharedMemory::wrapMap(const_cast<uint8_t*>(segment->data()), segment->size(), SharedMemory::Protection::ReadOnly);
+    RefPtr<SharedMemory> sharedMemory;// SharedMemory::wrapMap(const_cast<uint8_t*>(segment->data()), segment->size(), SharedMemory::Protection::ReadOnly);
```

Comment:
@kkinnunen-apple so this is one of the few places where `SharedMemory::wrapMap` is being used. How do we want to get change this callsite?